board.go

```diff
@@ -0,0 +1,70 @@
+package honeycombio
+
+// Compile-time proof of interface implementation.
+var _ Boards = (*boards)(nil)
+
+// Boards describes (some of) the board related methods that Honeycomb supports.
+type Boards interface {
+	// List all boards present in this dataset.
+	List() ([]Board, error)
+
+	// Get a board by its ID. Returns nil, ErrNotFound if there is no board
+	// with the given ID.
+	Get(id string) (*Board, error)
+
+	// Create a new board.
+	Create(b *Board) (*Board, error)
+
+}
+
+// boards implements Boards.
+type boards struct {
+	client *Client
+}
+
+// Board represents a Honeycomb board, as described by https://docs.honeycomb.io/api/boards-api/#fields-on-a-board
+type Board struct {
+	ID          string       `json:"id,omitempty"`
+	Name        string       `json:"name"`
+	Description string       `json:"description,omitempty"`
+	Style       string       `json:"style"`
```

Comment:
>Can we make a enum of this?

3f147c44f82a73efd7826fea21952e89f8f71ca9


>This field isn't required so it can also be set to omitempty I think.

010a4a9cd362e8fd97196f69c14ba782cf4beae9